boards/arm/warp7_m4/doc/warp7_m4.rst

```diff
@@ -0,0 +1,346 @@
++---------------+---------------------+--------------------------------+
+| I2C4_SDA      | I2C4_SDA            | I2C / FXOS8700 / FXAS21002     |
++---------------+---------------------+--------------------------------+
+| I2C4_SCL      | I2C4_SCL            | I2C / FXOS8700 / FXAS21002     |
++---------------+---------------------+--------------------------------+
+| SENSOR_INT_B  | ENET1_RD0/GPIO7_IO0 | FXOS8700 INT1 / FXAS21002 INT1 |
++---------------+---------------------+--------------------------------+
+
+System Clock
+============
+
+The M4 Core is configured to run at a 200 MHz clock speed.
+
+Serial Port
+===========
+
+The iMX7S SoC has seven UARTs. The number 2 is configured for the console and
+the number 6 is used in the mikroBUS connector.
+
+Programming and Debugging
+*************************
+
+The WaRP7 doesn't have QSPI flash for the M4 and it needs to be started by
+the A7 core. The A7 core is responsible to load the M4 binary application into
+the RAM, put the M4 in reset, set the M4 Program Counter and Stack Pointer, and
+get the M4 out of reset.
+The A7 can perform these steps at bootloader level or after the Linux system
+has booted.
+
+The M4 can use up to 5 different RAMs. These are the memory mapping for A7 and
+M4:
+
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| Region     | Cortex-A7             | Cortex-M4 (System Bus) | Cortex-M4 (Code Bus)  | Size                 |
++============+=======================+========================+=======================+======================+
+| DDR        | 0x80000000-0xFFFFFFFF | 0x80000000-0xDFFFFFFF  | 0x10000000-0x1FFEFFFF | 2048MB (less for M4) |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| OCRAM      | 0x00900000-0x0091FFFF | 0x20200000-0x2021FFFF  | 0x00900000-0x0091FFFF | 128KB                |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| TCMU       | 0x00800000-0x00807FFF | 0x20000000-0x20007FFF  |                       | 32KB                 |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| TCML       | 0x007F8000-0x007FFFFF |                        | 0x1FFF8000-0x1FFFFFFF | 32KB                 |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| OCRAM_S    | 0x00180000-0x00187FFF | 0x20180000-0x20187FFF  | 0x00000000-0x00007FFF | 32KB                 |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+| QSPI Flash |                       |                        | 0x08000000-0x0BFFFFFF | 64MB                 |
++------------+-----------------------+------------------------+-----------------------+----------------------+
+
+
+References
```

Comment:
I think this and the one at line 306 are not a heading since they are references only about the memory mapping and debugging.
Do you think I need to put as just a normal paragraph?

Reply:
@dbkinder,

Any suggestion?